miio/interfaces/vacuuminterface.py

```diff
@@ -21,3 +22,14 @@ def stop(self):
     def pause(self):
         """Pause cleaning."""
         raise RuntimeError("`pause` not supported")
+
+    @abstractmethod
+    def fan_speed_presets(self) -> Dict[str, int]:
```

Comment:
I think it'd be better to define `FanspeedPreset = Dict[str, int]` in this file, and use it to as type hint in other files.